sample_apps/run_example.sh

```diff
@@ -409,6 +679,88 @@ function print_signed_claim() {
    popd > /dev/null 2>&1
 }
 
+# ###########################################################################
+# Step 7 in instructions.md: Manual policy generation
```

Comment:
Step 7 here, and Step 8 (on L706) below refer to [contributions.md](https://github.com/vmware-research/certifier-framework-for-confidential-computing/blob/agurajada/67-run_example_for-oe-app/sample_apps/simple_app_under_oe/instructions.md). 

I need some help in better phrasing the section titles for these two steps in that md-file. Currently they read:

- Step 7: Manual policy generation
- Step 8: Use Policy Generator

I don't think these phrases convey the work that each type of this step is doing.

What is the difference between "Manual" and "Use Policy" ?
Can we come up with better section titles? I can then rework the comments / names of these functions in the shell script accordingly.

Reply:
Manually generate certifier policy or generate the same policy using the policy generator. Only one of these is required.

Reply:
Reworked names of functions and tags to:

- author_policy_for_OE() -> manual_policy_generation_for_OE()
- generate_policy_for_OE() -> automated_policy_generation_for_OE()
- Tag `setup_with_generate_policy` -> `setup_with_auto_policy_generation_for_OE`